### ISSUE TITLE
Work and artist search results should link to TA pages not archive

### ISSUE THREAD

**Issue by galenjosephhunter** [wavefarm/site#195]

**Comment by galenjosephhunter** [comment]
Right now artist archive returns include a "more at transmissionarts.org" link but work archive returns do not and are a dead-end. See:

https://wavefarm.org/archive/n2bna2
https://wavefarm.org/archive/azeqwc

**Comment by ptberry** [comment]
@galenjosephhunter  : I may have overstepped but yesterday I added code to the archive to link artists (w/categories:Transmission Artist) in archive search results to he individual artist page in /ta. That code is currently on dev. Artisrt w/o that category link to the archive page.

1) If that is correct, I think it supersedes the request above since users would never get to the individual artist page within the archive, right? It follows that the artists that remain in the archive should not have the "more at transmissionarts.org" at all.
2) About works, I was thinking archive work results should also link to /ta instead of the archive. Is this true? And if so, it is true for all works regardless of sites or categories?

**Comment by galenjosephhunter** [comment]
@ptberry Thank you for this. Yes, true for both artists categories:transmission artist and all works.

Would could simplify this considerable and simply say that for both artists and works if sites:transmission arts than they should link directly to the Transmission Arts Archive page.

Would that be preferable?

Galen

**Comment by ptberry** [comment]
Using "sites" instead of "categories" wherever possible does sound more 
intuitive to me, but the implementation is easy either way so it's 
really up to you in the end.

**Comment by galenjosephhunter** [comment]
Let’s go with being contingent on sites then, assuming Gabe is all fine with this?


Galen Joseph-Hunter
Executive Director
Wave Farm: Transmission Arts, WGXC 90.7-FM, Media Arts Grants
5662 Route 23 Acra, NY 12405 
518-622-2598 (o) | 917-297-1537 (c)
https://wavefarm.org/ <http://wavefarm.org/>
Sign-up for Wave Farm emails <http://eepurl.com/yRqVP>

**Comment by ptberry** [comment]
So this is just for artists and works, right? And do you want me to do 
this immediately  (on dev) ?

I'm not sure if you need to edit data first or not.

**Comment by galenjosephhunter** [comment]
Yes. Artists and Works only. This can be done immediately on production, unless you are worried about breaking something.

Galen Joseph-Hunter
Executive Director
Wave Farm: Transmission Arts, WGXC 90.7-FM, Media Arts Grants
5662 Route 23 Acra, NY 12405 
518-622-2598 (o) | 917-297-1537 (c)
https://wavefarm.org/ <http://wavefarm.org/>
Sign-up for Wave Farm emails <http://eepurl.com/yRqVP>

**Comment by ptberry** [comment]
I think this is done and can be closed. Do you agree @galenjosephhunter ?

**Comment by galenjosephhunter** [comment]
Actually looks like this maybe? is not fully working. It’s really weird actually at first load clicking on a work in the search results takes one to the archive page, if I refresh, the same link takes me to the Transmission Arts Archive. 

Is this a cache issue on my end or a site issue. 

There are definitely weird delay things happening with the site, for example see the “BROWSE BY SHOW NAME” link at https://wavefarm.org/wgxc/schedule <https://wavefarm.org/wgxc/schedule>

If you click on that link the search results say “0” for about 10 seconds and then switch to “115”.

Galen



Galen Joseph-Hunter
Executive Director
Wave Farm: Transmission Arts, WGXC 90.7-FM, Media Arts Grants
5662 Route 23 Acra, NY 12405 
518-622-2598 (o) | 917-297-1537 (c)
https://wavefarm.org/ <http://wavefarm.org/>
Sign-up for Wave Farm emails <http://eepurl.com/yRqVP>
